internal/helm/chart/builder.go

```diff
@@ -107,6 +107,8 @@ type BuildOptions struct {
 	// ValuesFiles can be set to a list of relative paths, used to compose
 	// and overwrite an alternative default "values.yaml" for the chart.
 	ValuesFiles []string
+	// Whether to silently ignore missing values files rather than failing.
```

Comment:
```suggestion
	// IgnoreMissingValuesFiles defines whether to silently ignore missing values files rather than failing.
```

Reply:
I semantically favour "controls" over "defines"… I hope you agree :-)